iceoryx_posh/include/iceoryx_posh/internal/popo/receiver_port.hpp

```diff
@@ -60,8 +60,8 @@ class ReceiverPort : public BasePort
     // BEGIN REGION__APPLICATION // /* access from Application-------------------------------
 
     virtual void subscribe(const bool f_autoResubscribe = false,
-                           const uint32_t f_deliverySize = MAX_RECEIVER_QUEUE_CAPACITY); // deprecated
-    virtual void subscribe(const uint32_t f_deliverySize = MAX_RECEIVER_QUEUE_CAPACITY);
+                           const uint32_t f_deliverySize = MAX_SUBSCRIBER_QUEUE_CAPACITY); // deprecated
```

Comment:
since it is deprecated, is it already possible to get rid of this method?

Reply:
receiver port will be deleted soon, when we switch to the new API

Reply:
Yeah, will be removed in the follow-up PR when #281 is merged.